### PR TITLE
Created single page app using React routing features.

### PR DESCRIPTION
**Pull request by ABDULALI3468**
### In this branch I did the following functionalities : 

1) I created a website consisting of 3 pages: Home, Calculator, and Quote.
2) I used React Router for navigation.
3) kindly review.
4) Thanks.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
-      <Calculator />
-    </div>
+    <BrowserRouter>
+      <div className="App">
+        <Nav />
+        <Routes>
+          <Route path="/home" exact element={<Home />} />
+          <Route path="/quote" element={<Quote />} />
+          <Route path="/calculator" element={<Calculator />} />
+        </Routes>
```
Comment (thecodechaser) [inline src/App.js]: - [ ] The Home component should be rendered on the root route. Right now if users will run your application they will see a blank page they need to click on the Home button to go to the homepage. Please fix this issue and make sure you call your Home component on the root route.
  - To fix this you can consider using this code `<Route path="/" exact element={<Home />} />`

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    textDecoration: 'none',
+  };
+
+  return (
+    <nav className="nav">
+      <h1>Math Magician</h1>
+      <ul className="nav-ul">
+        <Link style={navStyle} to="/home">
+          <li>Home</li>
+        </Link>
```
Comment (thecodechaser) [inline src/components/Navbar/Navbar.js]: - [ ] As I discussed above to use the root route for the Home component, You also need to fix your navbar. Please change the link for the Home button from `to="/home"` to this `to="/"`.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @ABDULALI3468👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Abdul, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ABDULALI3468** [comment]
Thank you so much @mikethreels and @thecodechaser for your support and guiding me.

**Pull request merged**